docs/examples/sujoy_edge/sujoy_edge_demo.jl

```diff
@@ -0,0 +1,80 @@
+# ---
+# cover: assets/sedge.png
+# title: Sujoy Edge Demo
+# ---
+
+using Images,Statistics,ImageView,TestImages
```

Comment:
No need for `ImageView` here.

```suggestion
using Images, Statistics, TestImages
```

Reply:
Fix this